### ISSUE TITLE
Lost ability to add Dates/ with base R

### ISSUE THREAD

**Issue by ldecicco-USGS** [USGS-R/gsplot#432]
Doing this:
```
gsplot() %>% points(1:10)
points(5,8, col="blue", pch=20)
```
produces 
![image](https://cloud.githubusercontent.com/assets/1105215/22792722/4cc1c010-eeb3-11e6-96b7-e659e211b1ca.png)

I would expect a blue dot on 5,8

It may have been broken awhile (?) because "unnamed-chunk-19-1.png" should have an extra dot somewhere (or we just didn't notice on a white -> transparent switch).

**Comment by ldecicco-USGS** [comment]
Seems like PR #431 is fixing this. Except with dates:

```
date_vector <- seq(as.Date("2010-10-01"), as.Date("2011-09-30"), by="months")
gs <- gsplot() %>% 
           points(date_vector, 1:12)
gs
points(as.Date("2011-01-15"),2.5, col="blue", pch=20)
```
![image](https://cloud.githubusercontent.com/assets/1105215/22797366/8e9e54d4-eec3-11e6-84a9-44294689c97e.png)


But if I use the index, then it's OK:
```
gs <- gsplot() %>% 
           points(date_vector, 1:12)
gs
points(3,2.5, col="blue", pch=20)
```
![image](https://cloud.githubusercontent.com/assets/1105215/22797396/b0a13984-eec3-11e6-8691-f76fbc917ecc.png)

I''m going to alter the title of this bug to reflect that.

**Comment by ldecicco-USGS** [comment]
Fixed in #431
